controllers/mock.go

```diff
@@ -341,10 +341,12 @@ func MockFrequentisChallenge(ctx context.Context, coduno *datastore.Key, w http.
 	taskTwo, err := model.Task{
 		Assignment: model.Assignment{
 			Name:        "AvlTree",
-			Description: "AvlTree",
-			Instructions: "Your task is to write Junit unit tests for an [AvlTree](https://en.wikipedia.org/wiki/AVL_tree) implementation.\n" +
+			Description: "Testing an Avl Tree",
+			Instructions: "Your task is to write tests for an [AvlTree](https://en.wikipedia.org/wiki/AVL_tree) implementation. You should be familiar with writing java unit tests using the [JUnit Framework](http://junit.org/)\n" +
```

Comment:
"AVL tree", "AvlTree" is just the class name, not the name of the thing in general.